ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -488,6 +489,16 @@ export default defineComponent({
       }
     )
 
+    watch(
+      () => hasUnsavedChanges.value,
+      (val: boolean) => {
+        if (val === false && context.refs.transferDetailsComponent) {
```

Comment:
Could probably use `if (!val...)` here to simplify.

Reply:
Updated.